cypress/e2e/views/common.view.ts

```diff
@@ -97,5 +96,19 @@ export enum sideDrawer {
     listText = "span.pf-v5-c-description-list__text",
     labelText = "span.pf-v5-c-label__text",
 }
-export const closeAbout = "button[aria-label='Close Dialog'";
+export const closeAbout = "button[aria-label='Close Dialog']";
 export const pencilIcon = "#pencil-action";
+
+// Task details page
+export const taskDetailsEditor = "div[class='pf-v5-c-code-editor__code']";
```

Comment:
Thanks for the PR ! 

Any reason why this is being moved out of taskmanager.view.ts ? Isn't this specific to the Task Manager page ?

Reply:
Thanks @nachandr 
I moved it to `common.view.ts` because you can access the task details view also from the application page (not only from Task Manager)